ui/dashboards/src/components/Dashboard/Dashboard.tsx

```diff
@@ -16,20 +16,21 @@ import { ErrorBoundary, ErrorAlert } from '@perses-dev/components';
 import { usePanelGroupIds } from '../../context';
 import { GridLayout } from '../GridLayout';
 import { EmptyDashboard, EmptyDashboardProps } from '../EmptyDashboard';
-
+import { PanelOptions } from '../Panel';
 export type DashboardProps = BoxProps & {
   /**
    * Props for `EmptyDashboard` component that will be rendered when the dashboard
    * is empty (i.e. has no panel groups). If not specified, the defaults will
    * be used.
    */
   emptyDashboardProps?: EmptyDashboardProps;
+  panelOptions?: PanelOptions;
```

Comment:
Future-looking question: do you think there's ever a case where a dashboard might want more granular panel options (e.g. by panel type or even at the individual panel level) or do you anticipate it always being global? I don't think it's worth spending time on a ton of complex premature optimization here, so mostly a curiosity question.

Reply:
Definitely, happy you thought about that as well. 

I left it out in this PR but I was thinking of injecting some context like the `panelDefinition` that would be all optional. Fortunately, we have some context hooks to get some info (eg. the TimeSeriesQuery through `useDataQueries()`) but I think we may need to inject some via args.

```tsx
type PanelOptions = {
 extra: (args: { panelDefinition?: PanelDefinition }) => React.Node
}
```